coins/bitcoin/interpreter/src/main/scala/co/ledger/lama/bitcoin/interpreter/Interpreter.scala

```diff
@@ -133,27 +134,32 @@ class DbInterpreter(
       ctx: Metadata
   ): IO[protobuf.ResultCount] =
     for {
-
       coin <- IO.fromOption(Coin.fromKey(request.coinId))(
         new IllegalArgumentException(s"Unknown coin type ${request.coinId}) in compute request")
       )
+      accountId           <- UuidUtils.bytesToUuidIO(request.accountId)
+      addresses           <- IO(request.addresses.map(AccountAddress.fromProto).toList)
```

Comment:
```suggestion
      addresses            = request.addresses.map(AccountAddress.fromProto).toList
```
You can also define it outside the for comprehension if I am not mistaken